gluon-mesh-vpn-wireguard-vxlan/files/lib/gluon/config-mode/wizard/0300-mesh-vpn.lua

```diff
@@ -0,0 +1,67 @@
+local unistd = require 'posix.unistd'
+
+local has_fastd = unistd.access('/lib/gluon/mesh-vpn/fastd')
+local has_tunneldigger = unistd.access('/lib/gluon/mesh-vpn/tunneldigger')
+local has_wireguard = unistd.access('/lib/gluon/mesh-vpn/wireguard')
```

Comment:
As this file overwrites the one provided by core gluon, i think it should be doable to abstract the VPN configuration state to the respective VPN packages.